openfish-webapp/.env

```diff
@@ -0,0 +1,2 @@
+# When testing locally use: http://localhost:3000
```

Comment:
Would this be port 8080 now?

Reply:
Yes, good catch. I have updated it here and elsewhere